gulpfile.babel.js

```diff
@@ -3,6 +3,7 @@ import {join} from 'path'
 import {gzipSync} from 'zlib'
 
 // Packages
+// eslint-disable-next-line no-unused-vars
 import _ from 'babel-polyfill'
```

Comment:
fwiw you can remove this import and `babel-polyfill` anywhere and install `transform-runtime`

```bash
npm install --save-dev babel-plugin-transform-runtime
```

then add it to the babel configuration as plugin:

```json
"babel": {
  "presets": [
    "es2015",
    "stage-3",
    "react"
  ],
  "plugins": [
    "transform-runtime"
  ]
}
```